File: app/src/main/java/com/stevenfrew/beatprompter/pref/SettingsActivity.java
package com.stevenfrew.beatprompter.pref;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

public class SettingsActivity extends AppCompatActivity
{
    SettingsFragment mFragment=new SettingsFragment();

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        // Display the fragment as the main content.
        getFragmentManager().beginTransaction()
                .replace(android.R.id.content, mFragment)
                .commit();
    }
}

File: app/src/main/java/com/stevenfrew/beatprompter/pref/SettingsActivity.kt
package com.stevenfrew.beatprompter.pref

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

class SettingsActivity : AppCompatActivity() {
    internal var mFragment = SettingsFragment()

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        // Display the fragment as the main content.
        fragmentManager.beginTransaction()
                .replace(android.R.id.content, mFragment)
                .commit()
    }
}
